stock_production_lot_firmware_version/__manifest__.py

```diff
@@ -4,16 +4,12 @@
 {
     "name": "Stock product lot firmware version",
     "summary": "Store the firmware version of a serialized electronic item",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "license": "AGPL-3",
     "author": "Open Source Integrators, Odoo Community Association (OCA)",
     "category": "Stock",
     "website": "https://github.com/OCA/product-attribute",
-    "depends": [
-        "stock"
-    ],
-    "data": [
-        "views/stock_view.xml",
-    ],
```

Comment:
Please avoid cosmetic changes.

Reply:
@bodedra FYI, Here Pre-commit suggest this kind of changes. That's why I changed code as per    [pre-commit ](https://travis-ci.org/OCA/product-attribute/jobs/601060152)

If not change then Travis is Failed.

Reply:
Ah. Good to know. Thank you.